src/IO/Observer/ReductionActions.hpp

```diff
@@ -229,6 +229,10 @@ void append_to_reduction_data(
     const gsl::not_null<std::vector<double>*> all_reduction_data,
     const std::vector<double>& t);
 
+void append_to_reduction_data(
+    const gsl::not_null<std::vector<double>*> all_reduction_data,
```

Comment:
clang-tidy wants you to delete `const` for this argument (and only this argument, because it has no effect in this function declaration as it's not part of a `const &` type declaration)